keycloak_realm_builder/scripts/custom_usage/tasks/bceid-prep.yml

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
Why is this file called "bceid-prep.yml"? Id doesn't do anything with bceid.

Reply:
This file is just a copy of sso-prep.yml so removing it.